public/locales/en.js

```diff
@@ -0,0 +1 @@
 you are using multiple wallet plugins. Please disable multiple wallet plugins, keep only one wallet plugin and try again.\",\"37qwvt\":\"Please download TokenPocket, connect your Dogecoin wallet within the app, and retry.\",\"bg/K/e\":[\"Please ensure that your browser has the \",[\"0\"],\" wallet plugin installed and try again.\"],\"KkRo4x\":\"Please open this page in your crypto wallet App and try again.\",\"s7sjPF\":\"Please switch your wallet to the BSC main network before connecting\",\"Iee1DF\":\"Please switch your wallet to the BSC Testnet before connecting\",\"ha3BZh\":\"Please switch your wallet to the Ethereum main network before connecting\",\"GEulUx\":\"Please switch your wallet to the Goerli test network before connecting\",\"Id0c/8\":\"Please switch your wallet to the Polygon Mainnet before connecting\",\"XM+H6v\":\"Please switch your wallet to the Polygon Testnet before connecting\",\"HV9LYa\":\"Please switch your wallet to the TRON main network before connecting\",\"oWL9at\":\"Please switch your wallet to the TRON Nile test network before connecting\",\"ZZQSW8\":\"Please try again after unlocking your TokenPocket wallet\",\"Lq3Ws8\":\"Please try again after unlocking your Tron wallet\",\"fuwKpE\":\"Please try again.\",\"toutGW\":\"QR Code\",\"6HTw2C\":\"replacement transaction underpriced\",\"QTIiXj\":\"Replacement transaction underpriced\",\"hW/LAG\":\"Request method eth_chainId is not supported\",\"GXsAby\":\"Revoke\",\"1wI7n7\":\"Revoke Failed\",\"zzTOEt\":\"Revoking...\",\"ovB7m2\":\"Scan QR Code\",\"/qjG3H\":\"Scan the QR code generated by your another device using this device, or paste the data here.\",\"l4+nu8\":[\"Scan with \",[\"0\"]],\"rtedPl\":\"Scan with your phone\",\"Tgqqaz\":\"Scanning QR code requires access to the camera. Please enable camera permission.\",\"TIC2wW\":\"Secure your assets by adding extra trusted devices like mobile phones, computers or tablets immediately.\",\"a1SmQh\":\"Select Wallet\",\"M7wPsD\":\"Switch\",\"ADQ2w8\":\"Switch Address\",\"emwlZI\":\"The contract verification failed.\",\"mncaot\":\"The current wallet environment does not support payments using TRX, please upgrade your wallet version or register with another wallet.\",\"au4g2p\":\"The new trusted device has full control over this address.\",\"l5QXS/\":\"the transaction has been aborted\",\"7wVond\":\"The transaction has been aborted.\",\"ohV77q\":\"This device or browser unsupported. Try connect by social or wallet.\",\"x4f8pZ\":\"Tips\",\"QGs22D\":\"transactionBuilder.addUpdateData\",\"VDX7Kz\":\"Trusted Devices of CKB Address\",\"LobdAW\":\"Unknown Error\",\"7P0626\":\"Unknown error happened. Please try again or paste the data directly.\",\"SoaXFN\":\"Use the QR code or link below on your other trusted devices and follow the instructions inside.\",\"OnLxXh\":\"User cancelled login\",\"B/NS7w\":\"user closed popup\",\"Hrrv+f\":\"User rejected the transaction\",\"jvvy/0\":\"user_canceled\",\"uny84M\":\"ValidationFailure(-31)\"}")};
```

Comment:
This directory and the following multi-language files are not useful to be deleted.

Reply:
Done